notebooks/official/gen_ai/tune_peft.ipynb

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        tuned_model_location="us-central1",  # Only us-central1 is supported
</pre><p> </p><p>Can this comment to move to the text block above? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>done</p>

Reply:
I meant moving this comment into the text block above the code block. 

```
tuning_job_location: The region where the tuning job should be run. Supported regions are: us-central1 and europe-west4.


tuned_model_location: The region where the tuned model should be deployed.
```